en/elasticsearch.rst

```diff
@@ -9,7 +9,7 @@ indexes easier.
 Installation
 ============
 
-To install the elasticsearch plugin, you can use ``composer``. From your
+To install the ElasticSearch plugin, you can use ``composer``. From your
```

Comment:
@dereuromark I have changed it in order to follow the pattern presented in the section of the DebugKit plugin and referred to ElasticSearch plugin through its name/namespace.